packages/types/src/index.ts

```diff
@@ -44,22 +44,28 @@ export interface ShowEvent {
 
 export interface ConnectEvent {
   is_atlas?: boolean;
+  resolved_hostname?: string;
   is_localhost?: boolean;
-  is_do?: boolean;
+  is_do_url?: boolean;
```

Comment:
Just checking – this is an intentional change to the data format we use for telemetry?

Reply:
We align this prom name with the rest of the dev tools.